docs: changelog: add mention of gateway file to v0.15 (#9231)
ipfs/kubo: docs/changelogs/v0.15.md

Extra plugin info [here](https://github.com/ipfs/kubo/blob/master/docs/plugins.md#fx-experimental).

#### 📁 `$IPFS_PATH/gateway` file

> This adds a new file in the `IPFS_PATH` folder similar to `$IPFS_PATH/api` containing a link to the current gateway.

Reviewer: Maybe mention that its based on the config?

```suggestion
This adds a new file in the `IPFS_PATH` folder similar to `$IPFS_PATH/api` containing an address based on [`Addresses.Gateway`](https://github.com/ipfs/kubo/blob/master/docs/config.md#addressesgateway) configuration.
```
